### PR TITLE
Update cef readme.md

### PR DESCRIPTION
**Pull request by cherryleaf**
Markdown tidy up

Type of change

- Enhancement

## What does this PR do?

Markdown tidy up

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by efd6 — APPROVED** [review]

File: packages/cef/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Common Event Format (CEF) Integration
 
 This is an integration for parsing Common Event Format (CEF) data. It can accept
-data over syslog or read it from a file.
+data over syslog or read it from a file.
```
Comment (efd6) [inline packages/cef/_dev/build/docs/README.md:4]: Why is this change necessary?

Comment (GinnyGalicia) [reply packages/cef/_dev/build/docs/README.md:4]: It was a way of adding the file to the PR.
